src/app/tree/node-tree/terra-node-tree.component.ts

```diff
@@ -118,17 +128,84 @@ export class TerraNodeTreeComponent<D> implements OnDestroy, OnInit
                 this.recursiveCheckList(node.children);
             }
         });
+
+        if(this._doExecuteFuzzySearch)
```

Comment:
Ist unter diesem If die komplette FuzzySearch?
Wenn ja, lager es am besten in eine eigene Funktion aus, welche du hier aufrufst.